bamboo/bootstrap-lint-audit.sh

```diff
@@ -15,4 +15,5 @@ if [[ $USE_CACHED_BOOTSTRAP == true ]]; then ## Change into cached cumulus, pull
 fi
 
 npm install --ignore-scripts --no-package-lock
+npx lerna run install-python-deps
```

Comment:
Where is `install-python-deps` defined?

Reply:
https://github.com/nasa/cumulus/pull/1521/files#diff-91d6cd211418b02ca2517245679dbe2eR19